### PR TITLE
feat: editor tab support

### PR DESCRIPTION
**Pull request by f97**
chrome-capture-2022-9-18

### CODE REVIEW

File: web/src/components/MemoEditor.tsx [hunk 1]
```diff
@@ -77,13 +77,16 @@ const MemoEditor: React.FC = () => {
...
-        handleFullscreenBtnClick();
-      }
-    } else if (event.key === "Enter" && (event.ctrlKey || event.metaKey)) {
+    if (event.key === "Escape" && state.fullscreen) {
+      handleFullscreenBtnClick();
+    }
+    if (event.key === "Enter" && (event.ctrlKey || event.metaKey)) {
       handleSaveBtnClick();
     }
+    if (event.key === "Tab") {
```
Comment (boojack) [inline web/src/components/MemoEditor.tsx:86]: `if-else` is more understandable, wdyt?

```suggestion
    } else if (event.key === "Enter" && (event.ctrlKey || event.metaKey)) {
      handleSaveBtnClick();
    } else if (event.key === "Tab") {
```

Comment (hyoban) [reply web/src/components/MemoEditor.tsx:86]: We need else if or return to avoid going into two branches

**Comment by f97** [comment]
I still think if is easier to understand, but it's fine for me.

**Review by hyoban — APPROVED** [review]
Sorry for taking your time, more people think `if` would be better

image

**Comment by f97** [comment]
@hyoban , Yeahhhh, I also just remembered I was missing `return`. Thanks....

**Pull request merged**